# Employment by Gender at UFRN

In [1]:
import pandas as pd
import numpy as np
from bokeh.charts import Donut, BoxPlot, show, output_notebook
from bokeh.layouts import row

df = pd.read_csv("csv/serversTreatedWithGender.csv")

In [2]:
genderDF = df
genderDF.head()

Unnamed: 0  Unnamed: 0.1  Unnamed: 0.1.1  \
0           0             0               0   
1           1             1               1   
2           2             2               2   
3           3             3               3   
4           4             4               7   

                                  name     money  \
0  adelardo adelino dantas de medeiros  17833.66   
1           abmael bezerra de oliveira  10388.52   
2             adailton garcia da silva   7464.42   
3               ada cristina scudelari  19995.43   
4                       adamo perrucci  10557.63   

                                       post class  level  \
0          PROFESSOR DO MAGISTERIO SUPERIOR     7  775.0   
1          PROFESSOR DO MAGISTERIO SUPERIOR     6  675.0   
2                   TECNICO EM AGROPECUARIA     D    NaN   
3          PROFESSOR DO MAGISTERIO SUPERIOR     8  800.0   
4  PROFESSOR MAGISTERIO SUPERIOR -VISITANTE     4  600.0   

                                       org gender  
0                PRO-REITORIA DE GRADUACAO      M  
1      DEPARTAMENTO DE ENGENHARIA ELETRICA      M  
2       ESCOLA AGRICOLA DE JUNDIAI - UAECA      M  
3         DEPARTAMENTO DE ENGENHARIA CIVIL      F  
4  DEPT DE DIREITO PROCESSUAL PROPEDEUTICA      M

## Overall Salary Distribution and Participation by Gender

In [3]:
def showStats(data):
    femaleDF = data[data.gender == 'F']
    maleDF = data[data.gender == 'M']
    print(str(len(data)) + " employees:")
    print("Males: " + str((len(maleDF)/len(data))*100) 
          + "%\tFemales: " + str((len(femaleDF)/len(data))*100) + "%")

    print("\nOverall salary:\nMedian: " + str(data['money'].median()) + "\tMean:" + str(data['money'].mean()))
    print("Male salary:\nMedian: " + str(maleDF['money'].median()) 
          + "\tMean:" + str(maleDF['money'].mean()))
    print("Female salary:\nMedian: " + str(femaleDF['money'].median())
          + "\tMean:" + str(femaleDF['money'].mean()))

    boxPlot = BoxPlot(data, values='money', label='gender',
                title="Salary Distribution by Gender", color = 'gender', plot_width=300, plot_height=500)
    donut = Donut(data, label=['gender'], title="Female and Male Servers", plot_width=500, plot_height=500)
    output_notebook()
    show(row([boxPlot, donut]))

showStats(genderDF)

5651 employees:
Males: 52.5747655282251%	Females: 47.3898425057512%

Overall salary:
Median: 7967.9	Mean:9236.546607679933
Male salary:
Median: 7648.76	Mean:9268.930912150801
Female salary:
Median: 8193.855	Mean:9204.896452576568


Loading BokehJS ...

The percentage of women in UFRN as a whole is close to that of men. The average salary for men is slightly higher. However, when checking the median salary, we see that women earn R$ 545 more than men. Why this difference between the mean and the median values?

### The Top 5%

One possibility to justify this would be to have more men earning high salaries because the median does not take into account the extreme values as the average does. To verify this, we will repeat the same analysis for the servers that earn the highest salaries in UFRN and see if there is a difference.

In [4]:
top = 5
quantile = genderDF.money.quantile(1.0 - top/100)
print("Top " + str(top) + "% salary >= R$" + str(quantile))
topDF = genderDF[genderDF.money >= quantile]
showStats(topDF)

Top 5% salary >= R$19912.21
284 employees:
Males: 63.38028169014085%	Females: 36.61971830985916%

Overall salary:
Median: 21889.394999999997	Mean:24501.595387323927
Male salary:
Median: 22045.915	Mean:24416.29144444445
Female salary:
Median: 21851.254999999997	Mean:24649.23682692307


Loading BokehJS ...

Now, looking at the top 5% better paid (with salaries greater than R $ 19912.21), there is a noticeable difference between the presence of men and women. Now, there are only 36 \% of women in this situation. Thus, it is observable that most high-level positions are being held by men. In addition, the mean and median wages of men are also higher than those of women.

# Conclusions 
In University world, besides being all politically "equal" (there is no difference between the processes of engagement to female or male functionaries), is clear that there is something to be observed when we realize that there is a big difference between the group of who is in top level spaces (earning greater salaries) and the rest.

The first plot show the greater outlines more frequentily in male group. But female mean salary is a little greater than male salary. Another fact is that there is more men in university than women.